integration-tests/kogito/javax.inject.Named

```diff
@@ -0,0 +1,4 @@
+org.drools.project.model.SessionRuleUnit_canDrinkKS
+org.drools.project.model.SessionRuleUnit_simpleKS
+io.quarkus.it.kogito.jbpm.OrdersProcess
+io.quarkus.it.kogito.jbpm.OrderItemsProcess
```

Comment:
This file shouldn't have been committed. The issue should be fixed with latest master.

Reply:
Sorry about that. Removed. Thanks.